File/contains crash during failed assertion

Quack,

When using the `contains` method I got this trace (only when the assertion fails):

```
    >       assert control_host.file(result_file).contains("Testing 123..")
    
    tests/test_default.py:30:
    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
    /home/duck/.local/lib/python3.7/site-packages/_pytest/assertion/rewrite.py:431: in _should_repr_global_name
        return not hasattr(obj, "__name__") and not callable(obj)
    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
    
    self = <testinfra.host.Host object at 0x7fb8628f3860>, name = '__name__'
    
        def __getattr__(self, name):
    >       assert name in testinfra.modules.modules, name + " is not a module"
    E       AssertionError: __name__ is not a module
    E       assert '__name__' in {'ansible': 'ansible:Ansible', 'command': 'command:Command', 'facter': 'puppet:Facter', 'file': 'file:File', ...}
    E        +  where {'ansible': 'ansible:Ansible', 'command': 'command:Command', 'facter': 'puppet:Facter', 'file': 'file:File', ...} = <module 'testinfra.modules' from '/home/duck/.local/lib/python3.7/site-packages/testinfra/modules/__init__.py'>.modules
    E        +    where <module 'testinfra.modules' from '/home/duck/.local/lib/python3.7/site-packages/testinfra/modules/__init__.py'> = testinfra.modules
    
    /home/duck/.local/lib/python3.7/site-packages/testinfra/host.py:105: AssertionError
```

I was used to the behavior the host/service module was providing:

```
            srv = host.service("cron")
    >       assert srv.is_running
    E       assert False
    E        +  where False = <service cron>.is_running
```

I tried to use file/content and pass a second argument as message to assert (which works for equality tests for example) but it ends-up with the same crash.

\\_o<

--- a/testinfra/host.py
+++ b/testinfra/host.py
@@ -102,11 +102,13 @@ class Host(object):
         return out.stdout.rstrip("\r\n")
 
     def __getattr__(self, name):
-        assert name in testinfra.modules.modules, name + " is not a module"
-        module_class = testinfra.modules.get_module_class(name)
-        obj = module_class.get_module(self)
-        setattr(self, name, obj)
-        return obj
+        if name in testinfra.modules.modules:
+            module_class = testinfra.modules.get_module_class(name)
+            obj = module_class.get_module(self)
+            setattr(self, name, obj)
+            return obj
+        raise AttributeError("'{}' object has no attribute '{}'".format(
+            self.__class__.__name__, name))
 
     @classmethod
     def get_host(cls, hostspec, **kwargs):

